src/corona/emissivity.jl

```diff
@@ -16,12 +16,20 @@ always considered to be the Keplerian velocity.
 
 Wilkins & Fabian (2012) and Gonzalez et al. (2017).
 """
-function source_to_disc_emissivity(m::AbstractStaticAxisSymmetric, N, A, x, g; Γ = 2)
+function source_to_disc_emissivity(
+    m::AbstractStaticAxisSymmetric,
+    spec::AbstractCoronalSpectrum,
+    N,
+    A,
+    x,
+    g,
+)
     v = CircularOrbits.fourvelocity(m, SVector(x[2], x[3]))
     # account for relativistic effects in area due to lorentz shift
     γ = lorentz_factor(m, x, v)
     # divide by area to get number density
-    N / (g^Γ * A * γ)
+    I = coronal_spectrum(spec, g)
+    N * I / (A * γ)
```

Comment:
NB: it makes more sense for coronal_spectrum to return the intensity, rather than the inverse of the intensity (I made the same mistake in the notes I wrote you but just flagging it here).